src/app/bundles/bundle-page/bundle-page.component.html

```diff
@@ -1,3 +1,26 @@
-<p>
-  bundle-page works!
-</p>
+<div class="govuk-grid-column-one-quarter">
+  <button type="submit" class="govuk-button">
+    Edit bundle details
+  </button>
+</div>
+<div class="govuk-grid-column-full">
+  <app-panel [title]=bundleTitle [description]=bundleDescription></app-panel>
+</div>
+<div class="govuk-grid-column-one-quarter">
+  <button type="submit" class="govuk-button">
+    Duplicate bundle
+  </button>
+</div>
+<div class="govuk-grid-column-one-quarter">
+  <button type="submit" class="govuk-button">
+    Archive bundle
+  </button>
+</div>
+<div class="govuk-grid-column-one-third">
+  <a class="govuk-button" [routerLink]="['/bundle','import',bundleId]">Add/Remove documents</a>
+</div>
+<div class="govuk-grid-column-full">
+  <bundle-documents [documents]="documents$ | async"
```

Comment:
What does `documents$ | async` do?

Reply:
| works in the same way as a pipe in linux, they're called pipes as well I think in angular
so the bit on the left is piped to the operator on the right
it is angular magic that basically waits for observables to resolve, then passes the value on

Reply:
thanks